tdrs-frontend/package.json

```diff
@@ -39,7 +39,6 @@
     "test:accessibility": "concurrently -k -s first 'REACT_APP_PA11Y_TEST=true npm run start:ci' 'wait-on http://localhost:3000/ && npm run pa11y-ci'",
     "test:e2e": "cypress open",
     "test:e2e-ci": "cypress run --headless -b chrome --env cypressToken=${CYPRESS_TOKEN}",
-    "test:e2e-develop": "cypress run --headless -b chrome --config baseUrl=https://${BASE_URL_PREFIX}-develop.app.cloud.gov --env cypressToken=${CYPRESS_TOKEN},apiUrl=https://${API_URL_PREFIX}-develop.app.cloud.gov/v1",
```

Comment:
This was meant to be a full integration test for the develop environment with all actual pieces (front-end, back-end, and all attached services) interacting, not testing each piece individually. Is there a replacement put in place I am not seeing?

Reply:
Yes, we will need to use test:e2e-ci, and use inline options to override urls, etc